lighthouse-core/report/html/renderer/category-renderer.js

```diff
@@ -292,17 +288,15 @@ class CategoryRenderer {
 
     const headerEl = this.dom.find('.lh-audit-group__header', clumpElement);
     const title = this._clumpTitles[clumpId];
-    headerEl.textContent = title;
+    this.dom.find('.lh-audit-group__title', headerEl).textContent = title;
     if (description) {
-      const markdownDescriptionEl = this.dom.convertMarkdownLinkSnippets(description);
-      const auditGroupDescription = this.dom.createElement('div', 'lh-audit-group__description');
-      auditGroupDescription.appendChild(markdownDescriptionEl);
-      clumpElement.appendChild(auditGroupDescription);
+      const descriptionEl = this.dom.convertMarkdownLinkSnippets(` — ${description}`);
+      descriptionEl.classList.add('lh-audit-group__description');
+      headerEl.appendChild(descriptionEl);
     }
 
     const itemCountEl = this.dom.find('.lh-audit-group__itemcount', clumpElement);
-    // TODO(i18n): support multiple locales here
-    itemCountEl.textContent = `${auditRefs.length} audits`;
+    itemCountEl.textContent = `(${auditRefs.length})`;
```

Comment:
i18n here too

Reply:
@exterkamp help

Reply:
This one looks A-Okay because the final output is all numeric, correct?  
"17 Audits" becomes "(17)" <- this should be fine.

The concern here is non latin numerics, like ar:
"(17)" -ar-> "(١٧)"

But afaik we already don't support that yet in LH or PSI consistently so I'd leave in the i18n TODO, but LGTM for now.